packages/e2e-test-kit/src/webpack-in-memory-test.ts

```diff
@@ -22,12 +16,30 @@ export function createMemoryFileSystemWithFiles(files: { [fullpath: string]: str
         memfs.writeFileSync(r, files[k] || '\n');
     }
 
+    if (includeStylableRuntime) {
+        /* 
+            TODO: load the runtime code explicitly in all test locations and remove this.
+        */
+        addStylableRuntimeToMemFs(memfs);
+    }
+
+    return memfs;
+}
+
+function addStylableRuntimeToMemFs(memfs: CustomMemoryFs) {
+    const runtimeDir = dirname(require.resolve('@stylable/runtime/cjs'));
```

Comment:
require.resolve once in root

Reply:
This is here because of the `laziness` of the operation that requires `@stylable/runtime` to be built and doing it in module scope can affect other places that dose not requires the runtime